charts/prometheus-node-exporter/README.md

```diff
@@ -41,6 +41,12 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### 4.8.x to 4.9.0
```

Comment:
i don´t think upgrading instructions are needed, as this is no major version change with breaking changes.
instead add this info above the podmonitor config please.

Reply:
Upgrade note removed, comment added to the configuration instead.